src/questions/javascript-questions.md

```diff
@@ -37,6 +37,7 @@ permalink: /questions/javascript-questions/index.html
   * How can you achieve immutability in your own code?
 * Explain the difference between synchronous and asynchronous functions.
 * What is event loop?
+* In which part of the HTML(Head,Body,footer) should we include the scripts ?
```

Comment:
what's the reason to add a question in the middle?

Reply:
Plus we have a question ```Why is it generally a good idea to position CSS <link>s between <head></head> and JS <script>s just before </body>? Do you know any exceptions?``` in html section

what's new gives your one?

Reply:
Putting CSS in body means it is loaded later. It is a technique some use to let the browser start drawing the interface faster (i.e., it removes a blocking step). This is important for user experience on SmartPhones. Source:Stack_overflow.
But , it wouldn't be good practice to include scripts in header before the HTML content is rendered(mostly) it can lead to slowdown of our site.

Reply:
I'm asking about the answer, my question is about your question

But even your answer is arguable